diff --git a/aiinpy.egg-info/PKG-INFO b/aiinpy.egg-info/PKG-INFO
index 139330c..7dce180 100644
--- a/aiinpy.egg-info/PKG-INFO
+++ b/aiinpy.egg-info/PKG-INFO
@@ -1,6 +1,6 @@
 Metadata-Version: 1.0
 Name: aiinpy
-Version: 0.0.5
+Version: 0.0.6
 Summary: A simple artificial intelligence library
 Home-page: UNKNOWN
 Author: UNKNOWN
diff --git a/dist/aiinpy-0.0.5.tar.gz b/dist/aiinpy-0.0.5.tar.gz
index 4bb6fd4..fa1614d 100644
Binary files a/dist/aiinpy-0.0.5.tar.gz and b/dist/aiinpy-0.0.5.tar.gz differ
diff --git a/examples/RNN_Example.py b/examples/RNN_Example.py
index 520dab9..db6d4f9 100644
--- a/examples/RNN_Example.py
+++ b/examples/RNN_Example.py
@@ -1,5 +1,8 @@
 import numpy as np
 import aiinpy as ai
+import wandb
+
+wandb.init(project='rnn')
 
 PositiveComments = open("C:\\Users\\smdro\\Downloads\\archive\\sentence_polarity\\rt-polarity.pos", "r")
 NegativeComments = open("C:\\Users\\smdro\\Downloads\\archive\\sentence_polarity\\rt-polarity.neg", "r")
@@ -7,15 +10,23 @@ Comments = str(PositiveComments.read() + NegativeComments.read())
 Comments = np.array(Comments.splitlines())
 
 UniqueWords = list(set([w for Sentence in Comments for w in Sentence.split(' ')]))
-Rnn = ai.RNN(len(UniqueWords), 2, "ManyToOne")
+Rnn = ai.RNN(8, 2, "ManyToOne")
+
+def WordToBinary(Input):
+  Dec = list(bytearray(Input, "utf8"))
+  Output = [''] * len(Input)
+  for i in range(len(Input)):
+    Output[i] = bin(Dec[i]).replace("b", ("0"*(9-len(bin(Dec[i])))))
+  return Output
 
-for Generation in range(1):
+for Generation in range(100000):
   Random = np.random.randint(0, len(Comments))
 
-  InputSentenceSplit = list(Comments[Random].split(' '))
-  Input = np.zeros((len(InputSentenceSplit), len(UniqueWords)))
-  for i in range(len(InputSentenceSplit)):
-    Input[i, UniqueWords.index(InputSentenceSplit[i])] = 1
+  t = WordToBinary(Comments[Random])
+  print(Comments[Random])
+  Input = np.zeros((len(t), 8))
+  for i in range(len(t)):
+    Input[i, :] = np.array(list(t[i]))
 
   Output = Rnn.ForwardProp(Input)
 
@@ -24,3 +35,4 @@ for Generation in range(1):
   OutputError = RealOutput - Output
 
   Rnn.BackProp(OutputError)
+  wandb.log({"Error": np.sum(np.abs(OutputError))})
\ No newline at end of file
diff --git a/setup.py b/setup.py
index 6986c0c..98bcb52 100644
--- a/setup.py
+++ b/setup.py
@@ -1,7 +1,7 @@
 from setuptools import setup
 
 setup(name='aiinpy',
-      version='0.0.5',
+      version='0.0.6',
       description='A simple artificial intelligence library',
       license='MIT',
       packages=['aiinpy'],
